src/Bicep.Core.Samples/Files/Metadata_CRLF/main.bicep

```diff
@@ -0,0 +1,45 @@
+/* 
+  This is a block comment.
+*/
+
+// metadata with value
+meta myString2 = 'string value'
+meta myInt2 = 42
+meta myTruth = true
+meta myFalsehood = false
+meta myEscapedString = 'First line\r\nSecond\ttabbed\tline'

```

Comment:
I don't think there should be any issues, but please could you also add a test to verify multi-line strings are working as expected?